android/src/com/google/zxing/client/android/CaptureActivity.java

```diff
@@ -221,7 +221,7 @@ private void resume(){
 
     SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(this);
 
-    if (prefs.getBoolean(PreferencesActivity.KEY_DISABLE_AUTO_ORIENTATION, true)) {
+    if (prefs.getBoolean(PreferencesActivity.KEY_DISABLE_AUTO_ORIENTATION, true) && intent.getBooleanExtra(Intents.Scan.PREVENT_ROTATION, true)) {
```

Comment:
Do we actually have a place in the code that sets the KEY_DISABLE_AUTO_ORIENTATION value in the shared preferences?

Reply:
no, but it was in there. The module had a properties UI where you could make those changes. There is a lot of code in there that shouldn't be in the module but in your own app, like checking for links etc (which makes is bloated). But that should be a different ticket (or a new module :) )

Reply:
I see. Let's leave it as is for the purpose of this PR then.